core/scss/components/organisms/_global-footer.scss

```diff
@@ -0,0 +1,15 @@
+@charset "UTF-8";
+
+//
+// Global Footer
+//
+// Global footer element as specified by the Identity Guidelines.
+//
+// Markup: ../../templates/components/organisms/global-footer/global-footer.twig
+//
+// Style guide: Organisms.Global Footer
+//
+
+.global-footer {
+  @include global-footer;
```

Comment:
I've put too much in to the mixin and should tease out the elements up and into this level.